scenarioo-server/src/main/java/org/scenarioo/rest/configuration/ConfigurationResource.java

```diff
@@ -41,7 +42,16 @@
 	public Configuration getConfiguration() {
 		return configurationRepository.getConfiguration();
 	}
-	
+
+	@GET
+	@Path("/diffViewer")
+	@Produces({ "application/json", "application/xml" })
+	public DiffViewerStatus getDiffViewerStatus() {
+		DiffViewerStatus status = new DiffViewerStatus();
+		status.setGraphicsMagickAvailable(GraphicsMagickConfiguration.isAvailable());
+		return status;
+	}
+
```

Comment:
That's the catch, there's none. Fulltext data is persisted into our configuration.